omnigibson/utils/usd_utils.py

```diff
@@ -823,7 +827,10 @@ def initialize_view(self):
 
         # Apply the pattern
         expected_prim_paths = {
-            prim_path for prim_path in expected_prim_paths if re.fullmatch(self._pattern.replace("*", ".*"), prim_path)
+            prim_path
+            for prim_path in expected_prim_paths
+            if re.fullmatch(self._pattern.replace("*", ".*"), prim_path)
+            or re.fullmatch(self._pattern.replace("*", ".*").split("/base_link")[0], prim_path)
```

Comment:
hmmm not sure I fully understand this. Also looping in @cgokmen and @hang-yin since I am not too familiar with this API.

Reply:
this is a really hacky fix for now, but I believe @cgokmen already has a more general fix for this and for the general /base_link constraint later